freqtrade/rpc/rpc.py

```diff
@@ -870,3 +870,6 @@ def _rpc_plot_config(self) -> Dict[str, Any]:
                 'subplots' not in self._freqtrade.strategy.plot_config):
             self._freqtrade.strategy.plot_config['subplots'] = {}
         return self._freqtrade.strategy.plot_config
+
+    def _rpc_sysinfo(self) -> Dict[str, Any]:
+        return {"cpu_pct": psutil.cpu_percent(interval=1, percpu=True), "ram_pct": psutil.virtual_memory().percent}
```

Comment:
Please either
* move this to the API file itself 
* keep it here, but make this method Static (use `_rpc_get_logs` as sample)

it's not accessing anything from within freqtradebot - so it can work fine as standalone function.
If your api call depends on rpc - it'll not work in webserver mode ... which would be a shame for this method.

Both above methods will work for this so it's up to you in the end.